gcp_variant_transforms/testing/data/vcf/bq_to_vcf/4_0_with_representative_header_file.vcf

```diff
@@ -0,0 +1,23 @@
+##fileformat=VCFv4.0
```

Comment:
nit: should this be under `...vcf/bq_to_vcf/expected_output/...` instead? We can have different representative header files or other arbitrary files here, so I think it's better to be explicit about which ones are about 'expected' output and which ones specify the input params.